images/linux/scripts/tests/Apt.Tests.ps1

```diff
@@ -14,6 +14,11 @@ Describe "Apt" {
             $toolName = "getfacl"
         }
 
+        if ($toolName -eq "aria2")
+        {
+        	$toolName = "aria2c"
+        }
```

Comment:
```suggestion
        if ($toolName -eq "aria2")
        {
            $toolName = "aria2c"
        }
```
nitpick: use spaces instead of tabs

Reply:
woops, did a quick edit in terminal and didn't noticed whitespace change, pushed fix